lib/dialects/mssql/data-types.js

```diff
@@ -1,179 +1,197 @@
 'use strict';
 
-var BaseTypes = require('../../data-types')
-  , util = require('util')
-  , _ = require('lodash');
+var _ = require('lodash');
+
+module.exports = function (BaseTypes) {
+  var warn = BaseTypes.ABSTRACT.warn.bind('https://msdn.microsoft.com/en-us/library/ms187752%28v=sql.110%29.aspx');
+
+  BaseTypes.DATE.types.mssql = [42];
```

Comment:
Each dialect specifies the ids / strings we should use to map to a parse function for that type